java-env.spec

```diff
@@ -1,22 +1,21 @@
 ### RPM external java-env 1.0
-## INITENV SET JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.131-2.b11.el7_3.x86_64
+## INITENV SET JAVA_HOME %{java_home}
+## INITENV +PATH PATH %{java_home}/bin
+## INITENV +PATH LD_LIBRARY_PATH %{java_home}/lib
+## NOCOMPILER
 
+%define java_home /usr/lib/jvm/java
```

Comment:
yes we need jdk-devel package and if that is missing the %{java_home}/bin/javac should fail